Semantics.md

```diff
@@ -663,7 +671,12 @@ round-to-nearest ties-to-even rounding.
 
 Truncation from floating point to integer where IEEE 754-2008 would specify an
 invalid operator exception (e.g. when the floating point value is NaN or
-outside the range which rounds to an integer in range) traps.
+outside the range which rounds to an integer in range) is handled as follows:
+ - For instructions with no exceptional behavior specified, a trap is produced.
+ - For instructions with the `:sat` modifier, no trap is produced, and:
```

Comment:
Exposing myself as not having a strong C/C++ background, is this min/max/zero functionality available as an intrinsic on mainstream processors?